toc/rfc/rfc-draft-hash-based-routing.md

```diff
@@ -0,0 +1,186 @@
+# Meta
+
+[meta]: #meta
+
+- Name: Hash-based routing
```

Comment:
one more thing: can we have the title a bit more extensive?

e.g. `Adding a hash-based load balancing algorithm for CF Routing` or something like that?

Reply:
I change the title.